app/services/medical_copays/vbs/request_data.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module MedicalCopays
+  module VBS
+    ##
+    # Object for handling VBS request parameters
+    #
+    # @!attribute user
+    #   @return [User]
+    # @!attribute edipi
+    #   @return [String]
+    # @!attribute vista_account_numbers
+    #   @return [Array]
+    # @!attribute errors
+    #   @return [Array]
+    class RequestData
+      attr_reader :user, :edipi, :vista_account_numbers
+      attr_accessor :errors
+
+      ##
+      # Builds a RequestData instance
+      #
+      # @param opts [Hash]
+      # @return [RequestData] an instance of this class
+      #
+      def self.build(opts = {})
+        new(opts)
+      end
+
+      ##
+      # The schema for validating attribute data
+      #
+      # @return [Hash]
+      #
+      def self.statements_schema
+        {
```

Comment:
Can the vets-json-schema repo be utilized for the schema validation?

Reply:
I don't know what that repo is 😳  I'll have to look into it, thanks! However, I think what I have going on right now is a bit heavy-handed for validating strings and could probably be simplified.